ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -18,24 +18,30 @@
 # ==================================================================================================
 
 
-def get_header(mrcfile: str) -> MrcInterpreter:
-    """Get the mrc file header for a tilt series without downloading the entire file."""
-    fs = FileSystemApi.get_fs_api(mode="s3", force_overwrite=False)
-    with fs.open(mrcfile, "rb") as f:
-        header = MrcInterpreter(iostream=f, permissive=True, header_only=True)
-    return header
+def get_header(mrcfile: str, fs: FileSystemApi) -> MrcInterpreter:
+    """Get the mrc file headers for a list of mrc files."""
+    try:
+        with fs.open(mrcfile, "rb") as f:
+            header = MrcInterpreter(iostream=f, permissive=True, header_only=True)
+        return header
+    except Exception as _:
+        pytest.fail(f"Failed to get header for {mrcfile}")
 
```

Comment:
Remove get_headers (only will ever get a header for one mrc_file) and adapt get_header